### DISCUSSION TITLE
Recovering transcriptions

### DISCUSSION THREAD

**Discussion by SionSR** [thuliteio/doks#1250] [How-to guides]
Everything looks to be working, except I can't find the transcribed files. What am I missing?

**Comment by h-enk** [comment]
Not sure what you mean by "transcriptions"?

Copy `node_modules/@hyas/doks-core/i18n/` over to `i18n/` and make your customizations in the latter.

**Comment by SionSR** [comment]
Thanks again Henk. The steps are a little different to what I understood.

I’ve got is sussed. YAY 😊 Thank you, Siôn
